modules/lvgl/lvgl_mem.c

```diff
@@ -10,7 +10,11 @@
 #include <zephyr/init.h>
 #include <zephyr/sys/sys_heap.h>
 
-static char lvgl_heap_mem[CONFIG_LV_Z_MEM_POOL_SIZE] __aligned(8);
+static char lvgl_heap_mem[CONFIG_LV_Z_MEM_POOL_SIZE]
+#ifdef CONFIG_LV_Z_MEMORY_POOL_CUSTOM_SECTION
+	Z_GENERIC_SECTION(.lvgl_heap)
+#endif
+		__aligned(8);
```

Comment:
This has some odd indentation, also you can use `IF_ENABLED`

```c
static char lvgl_heap_mem[CONFIG_LV_Z_MEM_POOL_SIZE]
	IF_ENABLED(CONFIG_LV_Z_MEMORY_POOL_CUSTOM_SECTION, (Z_GENERIC_SECTION(.lvgl_heap))
	__aligned(8);
```

Reply:
Good catch, changed it :^)

Reply:
Seem like it is not possible to use the `IF_ENABLED` macro here:
```
error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘IF_ENABLED’
```